contrib/core/actions/uuid.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: uuid
+runner_type: "python-script"
+description: Generate a new UUID (default uuid4)
+enabled: true
+entry_point: generate_uuid.py
+parameters:
+  uuid_type:
+    type: string
+    default: uuid4
```

Comment:
Small thing, should also add `enum` property:

```yaml

parameters:
  uuid_type:
    type: string
    default: uuid4
    enum:
      - uuid1
      - uuid4
```

This way other, non-valid values won't be accepted and a nice drop-down with possible values will be shown in the UI.